cernopendata/modules/theme/assets/semantic-ui/js/app.js

```diff
@@ -0,0 +1,3 @@
+import jquery from "jquery/dist/jquery";
+
+global.jquery = jquery;
```

Comment:
Yes, that's exactly right. Since CMS visualizer requires it's own (older) version of jQuery, those two versions conflict for the global `$` and `jQuery` namespaces. Since `glossary` started using `jquery` (lowercase) I thought we may reserve it globally for the Invenio jQuery version (left a comment there to explain why it is needed). So in this case when we load `/visualise/events/cms` page we have two versions working nicely:
```
> jquery().jquery
"3.5.1"
> $().jquery
"1.11.1"
> jQuery().jquery
1.11.1"
```